### PR TITLE
Jacob poker part

### PR DESCRIPTION
**Pull request by mcmillanjy**
Added the river class and find flush function

### CODE REVIEW

**Review by vondojr — APPROVED** [review]
Lots of files and code working here. Nice organization and seems to cover all bases. 

I'm wondering if we need to keep track of the highest card in a flush? If two players have a flush does it go to the high card? Not a problem either way though. 

Thanks for the comments!!!

File: source/River_Class.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from CardGames import *
```
Comment (JHerren726) [inline source/River_Class.py:1]: This class is a good start to the river functionality. I like it

File: source/Find_Flush.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    num_diamonds = 0
+    ### Need to have separate variables to keep track of the highest suit card
+    high_spade = Card("test", 0, [], [])
+    high_club = Card("test", 0, [], [])
+    high_heart = Card("test", 0, [], [])
+    high_diamond = Card("test", 0, [], [])
+    ### Add both Player's hand and the river
+    Cards= player.hand + river.hand
+    ### For every card, check to find the suit, then add 1 to the count
+    ### for that specific suit and see if that's the highest card in the suit
```
Comment (JHerren726) [inline source/Find_Flush.py:18]: I think you chose the best way to find the flush

**Review by JHerren726 — APPROVED** [review]
Great job!

File: source/River_Class.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+class River:
```
Comment (vondojr) [inline source/River_Class.py:3]: I think this will benefit quite a bit down the road...

File: source/Find_Flush.py (hunk 2 above)
Comment (mcmillanjy) [reply source/Find_Flush.py:18]: Thanks

File: source/River_Class.py (hunk 3 above)
Comment (mcmillanjy) [reply source/River_Class.py:3]: Yeah, I wanted to make sure I had everything to get started

**Pull request merged**